npm/cypress-schematic/src/e2e.spec.ts

```diff
@@ -25,7 +25,7 @@ const runCommandInProject = (command: string, projectPath: string) => {
 
 const cypressSchematicPackagePath = path.join(__dirname, '..')
 
-const ANGULAR_PROJECTS: ProjectFixtureDir[] = ['angular-17', 'angular-18']
+const ANGULAR_PROJECTS: ProjectFixtureDir[] = ['angular-18', 'angular-19']
```

Comment:
dropped 17 here for the schematic testing since it is time consuming and the schematic in 18 follows almost an identical pattern of that in 17